hw3/src/utils/chartColor.jsx

```diff
@@ -0,0 +1,45 @@
+export const backgroundColors = [
```

Comment:
I appreciate that you have this big blob of variables in a separate file, it makes `gdp.jsx` much cleaner